packages/cloud-functions/src/walletAddressMapping/index.ts

```diff
@@ -0,0 +1,23 @@
+import * as functions from 'firebase-functions'
+import { initDatabase, knexDb } from '../database/db'
+
+export const fetchAccountsForWalletAddress = functions.https.onRequest(async (req, res) => {
+  try {
+    res.set('Access-Control-Allow-Origin', '*')
+    const walletAddress = req.query.walletAddress as string
+    if (!walletAddress) {
+      res.status(400).end()
+      return
+    }
+    await initDatabase()
+    const accountAddresses = await knexDb('account_wallet_mappings')
```

Comment:
Who updates this table?

Reply:
The [indexer](https://github.com/valora-inc/indexer/)